presto-hive/src/main/java/com/facebook/presto/hive/authentication/KerberosHadoopAuthentication.java

```diff
@@ -51,4 +45,22 @@ public UserGroupInformation getUserGroupInformation()
         Subject subject = kerberosAuthentication.getSubject();
         return createUserGroupInformationForSubject(subject);
     }
+
+    private Configuration validateHDFSConfiguration(HdfsConfiguration hdfsConfiguration)
```

Comment:
`validate***` should return void (throwing on invalid state), or boolean indicating invalid state
Perhaps you want to pass `Configuration` here